pygitguardian/client.py

```diff
@@ -513,6 +514,54 @@ def create_honeytoken(
             result.status_code = resp.status_code
         return result
 
+    def create_honeytoken_with_context(
+        self,
+        *,
+        name: str,
+        type_: str,
+        description: Optional[str] = None,
+        project_extensions: list[str] = [],
```

Comment:
```suggestion
        project_extensions: List[str] = [],
```
from the typing module for the compatibility with Python < 3.9